example/lib/assurance.dart

```diff
@@ -0,0 +1,96 @@
+import 'dart:async';
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_aepcore/flutter_aepcore.dart';
+import 'package:flutter_aepassurance/flutter_aepassurance.dart';
+
+class assurancePage extends StatefulWidget {
```

Comment:
The dart code style mentions the [classes should use UpperCamelCase format ](https://dart.dev/guides/language/effective-dart/style#do-name-types-using-uppercamelcase) so can we rename all pages like this: AssurancePage, HomePage etc